GOVERNANCE.md

```diff
@@ -0,0 +1,96 @@
+  (which is open to all, and publicly logged).
+
+- **All decisions are made by community consensus.**
+  This does not mean there has to be unanimity,
+  nor that decisions result from vote counts.
+  What it means is that every interested member of the community can voice their thoughts,
+  and different positions are ideally resolved via
+  [informed consent](https://en.wikipedia.org/wiki/Sociocracy#Consent_vs._consensus)
+  of the involved people, who accept the collective decision
+  as "good enough for now, safe enough to try".
+
+
+## II. Role transitions
+
+The main goal of these principles is to support a continuous replenishing of the management team
+via a **smooth transition flow between community roles** —
+from newcomer, to occasional contributor, to regular contributor, to maintainer.
+This way the project can adapt in a flexible way
+to the the natural variations in availability and interest of its contributors,
+ensuring long-term resilience,
+and avoiding [single points of failure](https://en.wikipedia.org/wiki/Bus_factor).
+
+To this end, rather than assigning roles and tasks to people,
+these guidelines instead aim to **recognize the work that people already do**.
+Everyone is therefore encouraged to get involved and contribute to the project in whatever way they prefer,
+and we will strive to **get barriers out of the way** of these contributions.
+
+To ensure that these processes are transparent, predictable, and free from subective judgment,
+the metrics used are simple, objective, and publicly available.
+
+- Regular contributors shall be recognized as collaborators in the organization.
+
+    - Specifically: once a contributor has had **5 pull requests merged**,
+      they should be invited to become a
+      [**member of the tldr-pages organization**](https://github.com/orgs/tldr-pages/people).
+      This means they will be able to push commits to all of the organization's repositories,
+      merge PRs, label and close issues, among other things.
+      Note: All members of the tldr-pages organization must make their membership public.
+
+- Members of the organization who demonstrate interest in performing maintainership tasks,
+  by reviewing and/or merging PRs, responding to and labeling issues, and generally doing project maintenance work,
+  shall be made part of the maintenance team, and their name added to the list of current maintainers
+  in the [MAINTAINERS.md](MAINTAINERS.md) file.
+
+    - Specifically: once a contributor has been an organization member for at least 3 months,
+      and has **reviewed or merged 10 pull requests** by other contributors,
+      they should be invited to become an **owner of the tldr-pages organization**.
+      This means they will be able to add people to the organization,
+      manage all the organization's repositories, configure integrations, etc.
+
+- If a collaborator or maintainer stops being active in the project for more than 6 months,
```

Comment:
maybe state 'calendar months', so that people can easily look at their monthly activity on GitHub to determine when they need to do some more work in order to keep 'active'.  Ideally somehow phrase it so that an action on Jan 1 requires another action on July 31 at the latest.

Also I see one listed maintainer who this would apply to the moment the document becomes effective, in which case it would be nice to offer them the chance to fix the problem, either by doing an action, or stepping down so they are not listed in the maintainers document only to be immediately.

Reply:
I understand the sentiment, but I'm not sure "calendar months" would make the language clearer. For me at least, it would only make me more confused, as that's not a term I'm familiar with as a non-native English speaker.

In any case, the idea here is not to define activity requirements that people would have to fulfill in order to keep their membership. It's more of a way to allow the official list of maintainers to adjust to accurately reflect the actual maintenance team. Perhaps we could tweak the wording here to reinforce this sentiment?

Also, note that since becoming active again is sufficient to have the membership reinstated (as the initial thresholds had already been met the first time around), this shouldn't be a hassle for contributors who go through a period of reduced availability. If anything, it should make things easier for them, by making it official that they're not publicly identified as maintainers, as that can convey a sense of obligation -- which we want to avoid because it can quickly lead to burnout.

Reply:
"more than 6 months" means "6 months and one day"

"more than 6 [calendar months](https://www.merriam-webster.com/dictionary/calendar%20month)" means anywhere between 6 and 7 complete months (minus seconds) depending on when the first action occurred.

The intend is being able to use GitHub's monthly activity list to see which month the person was active and let them have six complete named months pass without activity before the pin is pull, without caring which day of the month their last activity was.

Reply:
> use GitHub's monthly activity list to see which month the person was active

Are you talking about the activity graph in the user profile? Perhaps this wasn't clear, but the intention here was to count activity in the tldr project, not on github overall. Or am I missing something?

> And moving people to inactive would be a regular first day of month maintenance task, not relative to when each persons last action day occurred.

That's a good point, but the idea was not to control each maintainer's activity down to such detail, but just to provide a rough indication of the period after which it would make sense to look at how long they've been effectively inactive in the project and then update the member list accordingly. If it takes a week or a month more to update the list, there's no harm. Besides, in our experience this shouldn't be a common enough occurrence to warrant establishing a monthly cleanup schedule.

That said, I think we can add an additional bullet point saying that they should get a friendly message informing them of the fact -- perhaps even provide a boilerplate message similar to [the one Homebrew provides for inviting maintainers](https://docs.brew.sh/New-Maintainer-Checklist.html), just to make sure that the removal won't give out the wrong impression.